src/researches/english/EnglishParticiple.js

```diff
@@ -82,25 +22,11 @@ var precedesIndex = function( precedingWords, participleIndex ) {
  */
 var EnglishParticiple = function( participle, sentencePart, attributes ) {
 	Participle.call( this, participle, sentencePart, attributes );
-	this.checkException();
+	checkException.call( this );
```

Comment:
Why this change?

Reply:
Before the change, the function `checkException` had been defined as a method of the `EnglishParticiple` object prototype. This function has now been moved to an external file because it’s also used for the French participle. In order to make the `this` in the `checkException` function definition refer to `EnglishParticiple`, `call( this )` was used to pass the correct reference for `this` (the current object) when calling the function.